frontend/lib/app.tsx

```diff
@@ -305,7 +312,8 @@ export class AppWithoutRouter extends React.Component<AppPropsWithRouter, AppSta
           <AppContext.Provider value={this.getAppContext()}>
             <AriaAnnouncer>
                 <Navbar/>
-                <MoratoriumBanner />
+                {ROUTES_WITH_MORATORIUM_BANNER.includes(this.props.location.pathname) 
+                  && <MoratoriumBanner />}
```

Comment:
Just so you know, one side-effect of conditionally displaying this component is that the state of the banner (open vs. closed) will be reset whenever it's unmounted and remounted... In effect that means that if I land on DDO and manually close the banner, it will re-appear when I click on one of the CTAs (since that's a splash page), which might be annoying.

To avoid this, you could _always_ render `<MoratoriumBanner>` but pass it `props.location.pathname` as a prop; then in _its_ render method/function, you could have it check to see if the passed-in pathname prop is one of the ones where it's actually supposed to appear, and if not, return `null`.  This means its open/closed state will be preserved for as long as the page lives, and the situation I described won't happen (at least not for JS-enabled users).

Anyways, it's just an enhancement, and I understand that this is pretty time-sensitive so it's definitely not a blocker!

Reply:
Oh wait!  Actually I don't think it will be unmounted/remounted when going from DDO to a splash page, since that won't result in an unmount/remount.  But if I e.g. click "get started" on the splash page but then click the back button--or if I go through the whole product flow and then click "take action" in the nav bar to go back to DDO--I think the banner would re-appear, which could be annoying.

Reply:
Interesting! I noticed that problem but didn't know of a workaround. I just implemented this and it seems to be working well on my end— lmk if it works for you too!